app/assets/javascripts/controllers/security_group/security_group_form_controller.js

```diff
@@ -12,11 +12,11 @@ ManageIQ.angular.app.controller('securityGroupFormController', ['$http', '$scope
     $scope.securityGroupModel.description = "";
     $scope.newRecord = true;
   } else {
-    miqService.sparkleOn();
-
-    $http.get('/security_group/security_group_form_fields/' + securityGroupFormId)
-      .then(getSecurityGroupFormData)
-      .catch(miqService.handleFailure);
+    API.get("/api/security_groups/" + securityGroupFormId + "?attributes=cloud_tenant").then(function(data) {
```

Comment:
@gildub Every API call needs to take into account the possibility of a failure. 
That's the reason we also use a `.catch` along with `.then`

The standard way of making an API call in our codebase is:
```javascript
 API.get(getUrl)
  .then(getFormData)
  .catch(miqService.handleFailure);
```

You will also find plenty of examples in the code.